### ISSUE TITLE
Typo in the avd plugins documentation

### ISSUE THREAD

**Issue by ghost** [aristanetworks/ansible-avd#3793]
In the [`ansible_collections/arista/avd/docs/plugins/Modules_and_action_plugins/eos_designs_structured_config.md`](https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/docs/plugins/Modules_and_action_plugins/eos_designs_structured_config.md?plain=1) file.

There is a small typo in the synopsis section `line 20` that reads `eos_design_facts` instead of `eos_designs_structured_config`.
```
The `arista.avd.eos_designs_facts` module is an Ansible Action Plugin providing the following capabilities:
```

**Comment by gmuloc** [comment]
Thanks for reporting! Fix in fligh in #3795 !

**Comment by ghost** [comment]
Yikes! not sure how I missed that
